### PR TITLE
Consistently use `npm` instead of `yarn` [Docs][Review]

### PR DESCRIPTION
**Pull request by giladronat**
Addresses #4514. @shannonbux @WebRuin feel free to check if I caught all instances.

## TODO

- [x] Go through `docs/`
	- [x] Should [CONTRIBUTING.md]() be changed to `npm`? It says “[it] relies on lerna and yarn to configure the repository...” so I don’t know if it should be touched. [No](https://github.com/gatsbyjs/gatsby/pull/4607#issuecomment-374565536)
	- [x] Should [GitLab deploy instructions](https://www.gatsbyjs.org/docs/deploy-gatsby/#build-and-deploy-with-gitlab-ci) be changed to `npm`? Not sure if GitLab relies on `yarn` or not, since the yml build script mentions `yarn install`. [Yes](https://github.com/gatsbyjs/gatsby/pull/4607#issuecomment-374565536)
- [x] Go through `packages/`
- [x] Go through `blogs/` -- Only one post had `yarn`-only instructions, changed [here](https://github.com/gatsbyjs/gatsby/pull/4607/commits/0985427663e1b9b74e1c2d090781194ae684353e)
- [x] Go through `examples/`
- [x] ~~Go through translated docs?~~ [Nah](https://github.com/gatsbyjs/gatsby/pull/4607#issuecomment-374565536)

### CODE REVIEW

**Comment by m-allanson** [comment]
> Should CONTRIBUTING.md be changed to npm? It says “[it] relies on lerna and yarn to configure the repository...” so I don’t know if it should be touched.

This should be left as `yarn` I think.

> Should GitLab deploy instructions be changed to npm? Not sure if GitLab relies on yarn or not, since the yml build script mentions yarn install.

I'm fairly sure it's ok to switch this to `npm`

**Comment by KyleAMathews** [comment]
Yeah, for our development, we use Yarn

**Comment by shannonbux** [comment]
@m-allanson and @KyleAMathews is this ready to merge? I self-assigned myself to review this and now realize I don't know enough to do it.

File: docs/docs/gatsby-style-guide.md [hunk 1]
```diff
@@ -83,13 +83,13 @@ repository.
...
 
-Finally, if you want to run a version of the guides repository locally, follow
+Finally, if you want to run a version of the Guides repository locally, follow
 these steps:
 
-1. Ensure you have the yarn package manager installed `npm install -g yarn`
-2. Install the Gatsby cli `yarn add --global gatsby-cli`
-3. Follow the steps above to fork and clone the Gatsby repo.
-4. Enter the `www` directory where gatsbyjs.org is and install its dependencies `yarn install` then run `gatsby develop`.
+1. Install the Gatsby command-line tool `npm install --global gatsby-cli`.
```
Comment (KyleAMathews) [inline docs/docs/gatsby-style-guide.md]: For our internal stuff, we do want people to use Yarn as we commit yarn.lock files.

**Comment by KyleAMathews** [comment]
@shannonbux left a review :-)

**Comment by giladronat** [comment]
TIL GitHub has a review and comment layer on top of just Git commits...
@shannonbux I reverted the change according to @KyleAMathews’s review, you have to approve it though for whatever reason (I can’t approve a change to my PR maybe?)
I’m working through the other rest of the list right now.

**Comment by giladronat** [comment]
I’m not touching ["Creating a Blog with Gatsby”](https://www.gatsbyjs.org/blog/2017-07-19-creating-a-blog-with-gatsby/) because he explains how to use `npm`, and it’s his blog post:
> We’re using yarn, but npm can just as easily be used with npm i --save [deps].

**Comment by giladronat** [comment]
Finished. Not sure how to tag under review, but it’s ready for review :)

**Comment by KyleAMathews** [comment]
Thanks!

**Pull request merged**